x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/connector_add_flyout.tsx

```diff
@@ -141,9 +142,9 @@ export const ConnectorAddFlyout = ({
           i18n.translate(
             'xpack.triggersActionsUI.sections.addConnectorForm.updateErrorNotificationText',
             {
-              defaultMessage: 'Failed to create connector: {message}',
+              defaultMessage: '{message}',
               values: {
-                message: errorRes.body?.message ?? '',
+                message: errorRes.body?.message ?? 'Cannot create a connector.',
               },
```

Comment:
This seems like a weird usage of i18n as it means you can't translate the 'Cannot create a connector.' message to any other language.

I think it should be something like:
```
.catch(errorRes => {
        toastNotifications.addDanger(
          errorRes.body?.message ??
          i18n.translate(
            'xpack.triggersActionsUI.sections.addConnectorForm.updateErrorNotificationText',
            { defaultMessage: 'Cannot create a connector.' }
          )
        );
      });
```
This means if there's a custom message - it'll show that.
Otherwise it'll show "Cannot create a connector." which can still be translated :)